miner/worker.go

```diff
@@ -396,7 +396,7 @@ func (miner *Miner) commitBlobTransaction(env *environment, tx *types.Transactio
 	// isn't really a better place right now. The blob gas limit is checked at block validation time
 	// and not during execution. This means core.ApplyTransaction will not return an error if the
 	// tx has too many blobs. So we have to explicitly check it here.
-	if (env.blobs+len(sc.Blobs))*params.BlobTxBlobGasPerBlob > params.MaxBlobGasPerBlock {
```

Comment:
If the issue is due to `sc == nil`, I prefer to handle sc directly with `if sc != nil {}`, as the code is self-explanatory and doesn’t need additional comments.

Reply:
Sorry, I don't think so, more safety checking is better than no checking.

Reply:
I mean **check**  ("handle" is misleading)`sc` directly (instead of `tx.BlobHashes()`).